src/main/org/codehaus/groovy/transform/AbstractASTTransformation.java

```diff
@@ -115,6 +115,14 @@ public int getMemberIntValue(AnnotationNode node, String name) {
         return 0;
     }
 
+    public boolean getMemberBoolValue(AnnotationNode node, String name) {

```

Comment:
I think it's common to use `memberHasValue` for evaluating boolean attributes, so this new method is probably not needed.